src/nbody/nbody.cpp

```diff
@@ -70,11 +72,12 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   if (integrator == "none") dt_reb = Big<Real>();
   params.Add("dt_reb", dt_reb);
 
-  // Unit system for gravity
-  const Real GM = pin->GetReal("gravity", "gm");
```

Comment:
Other packages like viscosity and maybe drag grab GM from here. So they'll need to be changed as well.

Reply:
`gm` is again a `param`